### ISSUE TITLE
Glide Typing will suggest sexual word

### ISSUE THREAD

**Issue by potatoes1286** [florisboard/florisboard#2150]
I've omitted the word from the title due to its nature.

#### Short description
I have not been able to find another sexual word which will be suggested via glide type.

However, the term "cumming" will be suggested via glide type. This can rarely be achieved on accident by glide typing "coming", or easily by accident glide typing "cunning". (I have not been able to replicate for "coming", but a friend has.)

I'm pretty sure this wasn't intentional.

#### Steps to reproduce
1. Enable Glide Typing
2. Intentionally try to glide type "cumming", or lazily glide type "cunning" or "coming"
3. Achieve suggestion "cumming"

#### Environment information
- FlorisBoard Version: 0.4.0-alpha04 (90)
- Install Source: F-Droid
- Device: Pixel 6
- Android: CalyxOS 4.5.1 (Android 13)

**Comment by BHydden** [comment]
Since they make you uncomfortable, I'll censor myself when I tell you you can glide type the F word and the C word 😅

Bug or feature? 🤷‍♂️ to each their own I guess

**Comment by potatoes1286** [comment]
True. There are a lot of people that may actually prefer with sexual or swears on.

Maybe it should be opt-in? It's been ages since I've used gboard or any other keyboard with type suggestion, but I don't believe they suggest sexual/swears by default.

**Comment by StepsisterVag** [comment]
I can confirm this issue.

My stepsister texted me 'can u hurry and bring me a clean towel 4 after my shower?'

I responded 'I'm cumming ASAP.' I was so embarrassed until she said, 'wait until you see me without the towel.'

So it can be seen as an actual keyboard feature and not a bug. Florisboard is bringing families together. Very close.

**Comment by patrickgold** [comment]
Thanks a lot your bug report!

Glide typing as of 0.4.0-alpha04 uses a hacked together word list which is pretty much unfiltered. As of 0.4.0-alpha05 this word list will not exist anymore and as such the unfiltered sexual and otherwise profane words will be much better filtered (except if you explicitly enable it). Attaching this issue to #2209 which will auto-close this issue once merged.
